docs/source/algorithms/DNSComputeDetEffCorrCoefs-v1.rst

```diff
@@ -81,20 +81,20 @@ Usage
    # Calculate correction coefficients
    coefs = DNSComputeDetEffCorrCoefs([vana_sf, vana_nsf], [bkgr_sf, bkgr_nsf])
 
-   print "First 3 correction coefficients: "
+   print("First 3 correction coefficients: ")
    for i in range(3):
-        print round(coefs.readY(i),2)
+        print(round(coefs.readY(i),2))
 
-   print "Is first detector masked?", coefs.getInstrument().getDetector(1).isMasked()
+   print("Is first detector masked?".format(coefs.getInstrument().getDetector(1).isMasked()))
```

Comment:
Is this correct? There are no format specifiers in the string. Should it be
`print("Is first detector masked? {}".format(coefs.getInstrument().getDetector(1).isMasked()))`

Reply:
Indeed!